frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/menu_user.dart

```diff
@@ -54,6 +60,49 @@ class MenuUser extends StatelessWidget {
     );
   }
 
+  Widget _renderThemeToggle(BuildContext context) {
+    final themeProvider = Provider.of<ThemeProvider>(context);
+    final provider = Provider.of<ThemeProvider>(context, listen: false);
+    final theme = context.watch<AppTheme>();
+    return Switch.adaptive(
+      value: themeProvider.isDarkMode,
+      onChanged: (value) {
+        final provider = Provider.of<ThemeProvider>(context, listen: false);
+        provider.toggleTheme(value);
+        print(themeProvider.isDarkMode);
+      },
+    );
+    // return Material(
+    //     // width: 25,
+    //     // height: 25,
+    //     // backgroundColor: Color.fromRGBO(132, 39, 224, 1.0),
+    //     child: CircleAvatar(
+    //   child: IconButton(
+    //       icon: Icon(themeProvider.isDarkMode ? Icons.dark_mode : Icons.light_mode),
+    //       color: (theme.shader1),
+    //       onPressed: () {
+    //         provider.toggleTheme(themeProvider.isDarkMode);
+    //         print(themeProvider.isDarkMode);
+    //         print(themeProvider.themeMode.name);
+    //         print(theme.textColor);
+    //       }),
+    // ));
+  }
+
+  Widget _renderDarkMode(BuildContext context) {
+    return Tooltip(
+      message: "Set to Dark Mode",
```

Comment:
Language support. 

e.g. LocaleKeys.subscribeNewsletterText.tr(),

Reply:
Noted will add that in.